go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/Azure/go-autorest/autorest v0.9.0
 	github.com/Azure/go-autorest/autorest/date v0.2.0
 	github.com/google/uuid v1.3.0
-	github.com/microsoft/moc v0.11.0-alpha.2
+	github.com/microsoft/moc v0.10.29-alpha.3
```

Comment:
Are we going back on the versioning?

Reply:
Ragha said he was using this tag for his own testing. I think v0.10.29-alpha.3 is the latest tag

Reply:
Please create something like v0.11.0-alpha.3 and use that as latest.
v0.10 is old

Reply:
Resolved